data/teca_calendar_util.h

```diff
@@ -375,7 +413,7 @@ class TECA_EXPORT interval_iterator_factory
 public:
     /** Allocate and return an instance of the named iterator
      * @param[in] interval Name of the desired interval iterator. One of daily,
-     *                     monthly, seasonal, or yearly
+     *                     monthly, seasonal, yearly, or n_steps
```

Comment:
Shouldn't line 422 include something like `n_steps = 6` in the enum?

Reply:
Yes, it probably should. The issue with how I did this on the Python side was to get the number of steps as a string that could be parsed eg: "5_steps". That needs to change in order wo make use of the enum.